.shellcheckrc

```diff
@@ -1,7 +1,6 @@
 # until https://github.com/checkstyle/checkstyle/issues/11637
 
 disable=SC2207 # (warning): Prefer mapfile or read -a to split command output.
-disable=SC2002 # (style): Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
```

Comment:
@romani I am good for permanent suppression of this rule, what do you think? I like left to right flow (via pipe) of logic in bash scripts.

Reply:
The normal way is very simple and good. But, wikipedia says it is abuse of ``cat``:

![image](https://user-images.githubusercontent.com/90137881/225089017-60dc06f2-8f6e-4962-8659-b3cb949806ad.png)

Reply:
@shubh220922 that's ok, wikipedia doesn't have to maintain our project ;) 

>The normal way is very simple and good

This is a sufficient reason for me to do something :)

Reply:
Done